### PR TITLE
chore(custom-resources): ignore certain outputs from env stack in env controller

### PR DESCRIPTION
**Pull request by Lou1415926**
Addressing https://github.com/aws/copilot-cli/pull/3771#discussion_r922325007
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

**Review by iamhopaul123 — APPROVED** [review]

File: cf-custom-resources/lib/env-controller.js [hunk 1]
```diff
@@ -308,7 +310,9 @@ const updateAliases = function (cfnAliases, workload, aliases) {
 const getExportedValues = function (stack) {
   const exportedValues = {};
   stack.Outputs.forEach((output) => {
-    exportedValues[output.OutputKey] = output.OutputValue;
+    if (!ignoredEnvOutputs.has(output.OutputKey)) {
+      exportedValues[output.OutputKey] = output.OutputValue;
+    }
```
Comment (efekarakus) [inline cf-custom-resources/lib/env-controller.js]: nit:
```suggestion
    if (ignoredEnvOutputs.has(output.OutputKey)) {
      return
    }
    exportedValues[output.OutputKey] = output.OutputValue;
```

File: cf-custom-resources/lib/env-controller.js [hunk 2]
```diff
@@ -16,6 +16,8 @@ const updateStackWaiter = {
 
 const AliasParamKey = "Aliases";
 
+let ignoredEnvOutputs = new Set(["EnabledFeatures", "LastForceDeployID"]);
```
Comment (efekarakus) [inline cf-custom-resources/lib/env-controller.js:22]: Can you add a comment w a link to the 4096 bytes limit and why we're ignoring it here? 🥺

**Review by efekarakus — COMMENTED** [review]
Awesome, ty!

**Review by efekarakus — APPROVED** [review]

File: cf-custom-resources/lib/env-controller.js [hunk 3]
```diff
@@ -16,6 +16,11 @@ const updateStackWaiter = {
 
 const AliasParamKey = "Aliases";
 
+// Per the doc at https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/crpg-ref-responses.html
+// The size of the response body should not exceed 4096 bytes.
+// Therefore, we should ignore any outputs that we don't need. g
```
Comment (dannyrandall) [inline cf-custom-resources/lib/env-controller.js]:
```suggestion
// the size of the response body should not exceed 4096 bytes.
// Therefore, we should ignore any outputs that we don't need.
```

**Review by dannyrandall — APPROVED** [review]

**Pull request merged**